warehouse/oidc/models/github.py

```diff
@@ -149,6 +145,12 @@ def job_workflow_ref(self):
     def sub(self):
         return f"repo:{self.repository}"
 
+    def publisher_url(self, claims=None):
+        sha = claims.get("sha") if claims else None
+        suffix = f"/commit/{sha}" if sha else ""
+
+        return f"https://github.com/{self.repository}{suffix}"
```

Comment:
Avoiding ternaries where they aren't necessary:

```suggestion
        base = f"https://github.com/{self.repository}"
        sha = claims.get("sha") if claims else None
        
        if sha:
            return f"{base}/commit/{sha}"
        return base
```